automatic/freecad/tools/helper.ps1

```diff
@@ -7,7 +7,7 @@ param(
 )
 $fileName = ((($thePackage.url64 -split('/'))[-1]) -replace( "\.$($thePackage.fileType)", '' ) )
  if ($fileName -match "portable") { 
-	if (Get-Command Get-Version){ $version = ( Get-Version $fileName ).Version;
+	if (Get-Command Get-Version -ErrorAction "Continue"){ $version = ( Get-Version $fileName ).Version;
```

Comment:
`Get-Version` is AU function that has no relevance to this package. You want  something like `Get-Item  $filePath | % VersionInfo | % FileVersion`

Reply:
@majkinetor 
Yes, this is a AU function. This is a Check to see if AU is installed on the system, and if so use the `Get-Version` function.  The package of FreeCAD _does not_ need **AU** as a **_dependency_**. If you want me to eliminate the `Get-Version` check I will do that option.

Reply:
Yes I know but its completelly wrong. Get-Version doesn't do that. Not sure it got there.